resources/views/layouts/app.blade.php

```diff
@@ -33,9 +33,9 @@
         @stack('scripts')
     </head>
     <body
-        class="@if(Settings::usesDarkTheme()) dark @endif @if(Settings::usesCompactTables()) table-compact @endif"
+        class="table-compact @if(Settings::usesDarkTheme()) dark @endif @unless(Settings::usesCompactTables()) table-compact-until-md @endunless"
```

Comment:
In case this is confusing is because, if you think about it, the table is **always** compact on < md screens, in explorer most of the tables (but not all) are lists in mobile but the ones that are not (like the ones on the `/delegates` page) are "compact" on small screens even if the expanded option is set.

So basically I am keeping the `table-compact` always but  and when the `useContactTables` is not set I am only using compact until md screens